test/system/example.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { test, expect } from '@playwright/test';
+
+test.describe(`Playwright Tests - @nextjs @nuxt @vanilla`, () => {
+    test('has title', async ({ page }) => {
+        await page.goto('https://playwright.dev/');
+      
+        // Expect a title "to contain" a substring.
+        await expect(page).toHaveTitle(/Playwright/);
+      });
+      
+      test('get started link', async ({ page }) => {
```

Comment:
```suggestion
      test('should show "Get started" link', async ({ page }) => {
```

Reply:
This test will get deleted in a follow-up PR. It's just here to prevent CI from falling over.